assets/findify-recommendation.css

```diff
@@ -114,4 +109,27 @@
     width: 98%;
     height: 1.6em;
     white-space: nowrap;
+}
+
+.swiper-button-next,
+.swiper-button-prev {
+    background-color: #4d89d5;
+    border-radius: 50%;
+    padding: 10px 20px;
+    color: white;
+    width: 40px !important;
+    height: 40px !important;
+}
+
+.swiper-button-next:after,
+.swiper-button-prev:after {
+    font-size: 10px !important;
+    color: white;
+    font-weight: 700;
+}
+
+.adapt {
+    width: 100%;
+    aspect-ratio: 1/1.5;
+    object-fit: cover;
```

Comment:
.adapt class probably should be within product-card.css ?

Reply:
done)